pkg/model/dependencies.go

```diff
@@ -0,0 +1,42 @@
+package model
+
+type dependenciesRaw struct {
+	Repository   string      `json:"repository,omitempty" yaml:"repository,omitempty"`
```

Comment:
should we support `name` and `wait` too?

Reply:
`name` is being defined when declaring the dependency map at the manifest. Should we add a name to override this map key value?
i will add the wait

Reply:
ah, makes sense